### ISSUE TITLE
AVT 3 - Tag Component React & Vanilla: JAWS and NVDA read the tags as one line with no spacer.

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2738]
**Environment:**
Windows 10
JAWS 19.1810.64
Chrome: 73.0.3683.103 (Official Build) (64-bit)
NVDA 2019.1
Carbon X React

**JAWS:**
When arrowing to the tags, JAWS reads the tags, but as one line with no spacer between tags. All tags are read as one word, there is no way to tell where one tag ends and where the next starts.

**NVDA**
When arrowing to the tags, NVDA reads the tags, but as one line with no spacer between tags. All tags are read as one word, there is no way to tell where one tag ends and where the next starts

**Comment by snidersd** [comment]
@dakahn Please add the react label to this defect. Thx!

**Comment by snidersd** [comment]
@dakahn This issue needs to be added to the[ IBM Carbon Copy Milestone](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+label%3A%22type%3A+a11y+%E2%99%BF%22+milestone%3A%22IBMa+Carbon+Copy+%F0%9F%A7%81%22)

**Comment by snidersd** [comment]
@dakahn The issue is due to a lack of structure around the tag. The recommendation is to change this to a list (role=list)  with each tag as a list item. We have confirmed this solution will work with JAWS and NVDA for both React and Vanilla.

**Comment by dakahn** [comment]
@snidersd Which set of tags are we testing against? The example on our React site is a single tag. Totally agree that if we had multiple tags they should be in an unordered list though!

**Comment by snidersd** [comment]
@dakahn I tested the React code but included multiple tags since this is a common practice and consistent with the Vanilla example.

**Comment by dakahn** [comment]
@snidersd Oh gotcha! I'll update the two vanilla/React examples with `<ul>` 👍
